src/app/components/account/account-avatar/account-avatar.tsx

```diff
@@ -1,47 +1,51 @@
-import { memo, Suspense } from 'react';
+import { memo } from 'react';
 import { BoxProps, color, Circle } from '@stacks/ui';
 
 import { useAccountGradient } from '@app/common/hooks/account/use-account-gradient';
+import { AccountWithAddress } from '@app/store/accounts/account.models';
+import { useAccountDisplayName } from '@app/common/hooks/account/use-account-names';
+
+const getAvatarText = (name: string, account: AccountWithAddress) => {
```

Comment:
Renamed this function because the previous name implied that it could only return a single char/grapheme, however when the users have more than 9 accounts, a double char string generated from the account's number may be returned.